mcc/GetSetAttrs.c

```diff
@@ -413,9 +413,11 @@ IPTR mSet(struct IClass *cl, Object *obj, struct opSet *msg)
 
               ScrollRasterBF(data->rport, 0, smooth,
                             _mleft(obj), _mtop(obj),
-                            _mright(obj), _mtop(obj) + (data->maxlines * data->fontheight) - 1);
+                            _mright(obj), _mbottom(obj));
+
               data->ypos = _mtop(obj) - ti_Data%data->fontheight;
               lines = (-smooth/data->fontheight)+2;
+
               {
                 struct Layer *layer = data->rport->Layer;
```

Comment:
Please move the definition of `layer` to line 412 and remove the additional { } block so that indentation is preserved.